cmd/zc_traverser_blob.go

```diff
@@ -478,6 +478,8 @@ func (t *blobTraverser) createStoredObjectForBlob(preprocessor objectMorpher, bl
 		object.blobSnapshotID = common.IffNotNil(blobInfo.Snapshot, "")
 	} else if t.includeDeleted && t.includeVersion && blobInfo.VersionID != nil {
 		object.blobVersionID = common.IffNotNil(blobInfo.VersionID, "")
+	} else if blobInfo.VersionID != nil {
+		object.blobVersionID = common.IffNotNil(blobInfo.VersionID, "")
```

Comment:
Does this add version id in cases we might not want it?

Reply:
That's a good point, I updated the code so that we only add VersionId for list commands if it is mentioned by the user. Let me know what you think!